### DISCUSSION TITLE
Help for Mathjax integrating to dify chat clent, rendering Mathematical formulas

### DISCUSSION THREAD

**Discussion by huangzewen** [langgenius/dify#8637] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

Help for Mathjax integrating to dify chat clent, rendering Mathematical formulas。

For example： $$ E = mc^2 $$

**Comment by huangzewen** [comment]
Is this request related to a challenge you're experiencing? Tell me about your story.
Help for Mathjax integrating to dify chat clent, rendering Mathematical formulas。

For example： $$ E = mc^2 $$

**Comment by Fucheng-Wu** [comment]
same issue
